packages/event-handler/src/types/index.ts

```diff
@@ -8,3 +8,22 @@ export type {
   RouteOptions,
   RouterOptions,
 } from './appsync-events.js';
+
```

Comment:
I'm not sure what the difference for exporting types here and the `packages/event-handler/src/types/bedrock-agent-function.ts` file is here so I've exported everything.

Reply:
Yeah, makes sense. I've removed all the types that are only being used internally.